### PR TITLE
New spec for getting oracle asm disks udev rules

### PR DESCRIPTION
**Pull request by qinpingli**
### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This spec is used to detect the Oracle ASM diskgroup outage issue proactively. If the disks in the Oracle ASM diskgroup are managed by udev, and the `add|change` actions are not configured `nowatch` in the udev rule, the customer will hit this issue. Detailed info is in the CEECBA-5525.

### CODE REVIEW

**Review by xiangce — APPROVED** [review]
Approved pending the approval of the Spec.

**Comment by bfahr** [comment]
@qinpingli please complete the description section of your PR so that we can properly evaluate your PR.

**Comment by qinpingli** [comment]
@bfahr Updated the description part, please help take a look again, thanks!

**Comment by qinpingli** [comment]
@bfahr when you get some time, could you help review this PR, thanks in advance!

File: insights/specs/__init__.py
Comment (bfahr) [inline insights/specs/__init__.py:149]: @qinpingli this spec is a `glob_file` which means that it can collect multiple inputs, so you'll need to add the attribute `multi_output=True`.

**Review by bfahr — CHANGES_REQUESTED** [review]
Please make the requested change and we can re-review once the spec has been approved.

File: insights/specs/__init__.py
Comment (qinpingli) [reply insights/specs/__init__.py]: @bfahr Good catch, thanks! Updated.

File: insights/parsers/etc_udev_rules.py
Comment (ryan-blakley) [inline insights/parsers/etc_udev_rules.py:61]: @qinpingli based on #2897 could you move the new parser to the udev_rules parser instead of the etc_udev_rules parser?

Comment (qinpingli) [reply insights/parsers/etc_udev_rules.py:61]: @ryan-blakley Do you mean all the dev rules in `/lib/udev/rules.d` and `/etc/udev/rule.d` dirs should be placed in `udev_rules` based on `All udev rules related parser can be placed in udev_rules module only.`?

Yes, I saw the 40-redhat udev rule was moved to the `udev_rules`, I thought the reason is 40-redhat udev rule is in `/lib/udev/rules.d` dir.

Comment (ryan-blakley) [reply insights/parsers/etc_udev_rules.py:61]: @qinpingli yes, since udev rules can be in multiple locations, I think it's more logical to place all udev rule parsers under the udev_rules module instead of splitting them between two modules based on the location.

Comment (qinpingli) [reply insights/parsers/etc_udev_rules.py]: @ryan-blakley got it, thanks! Moved the parser to `udev_rules`, please help take look again, thanks!

Comment (ryan-blakley) [reply insights/parsers/etc_udev_rules.py]: @qinpingli Awesome thank you, everything looks good to me.

**Comment by lhuett** [comment]
@qinpingli Before we can merge this we need the insights-core-assets spec (MR #254) to be approved which is waiting on the rule to be merged.

**Comment by chenlizhong** [comment]
@lhuett  Rule testing is failed as this parser is needed.  Please merge this first.

**Pull request merged**
